package.json

```diff
@@ -8,7 +8,7 @@
     "test": "jest",
     "coverage": "jest --coverage",
     "storybook": "start-storybook -p 9001 -c .storybook",
-    "build:storybook": "build-storybook -c .storybook -o docs"
+    "build:storybook": "build-storybook -c .storybook -o docs && git add docs"
```

Comment:
good call

Reply:
Kevin and I were chatting about actually letting travis to this, which would make this more transparent (and reliable?)...  so hopefully this wont be needed in the future.  I am not sure I like getting files automatically staged in my current branch, but it seems like this is better than having to remember to check in the built artifacts. :)